src/nodedef.cpp

```diff
@@ -112,6 +160,26 @@ void NodeBox::deSerialize(std::istream &is)
 		wall_side.MinEdge = readV3F1000(is);
 		wall_side.MaxEdge = readV3F1000(is);
 	}
+	else if (type == NODEBOX_CONNECTED)
+	{
+#define READBOXES(box) \
+	do {	count = readU16(is); \
```

Comment:
It's technically not needed, but good macro coding practice when writing multi-line macros.